_includes/manuals/nightly/1.2_release_notes.md

```diff
@@ -4,6 +4,11 @@ This section will be updated prior to the next release.
 
 ### Headline features
 
+#### Array, Boolean and other types support in Parameters
+
+Not only string but array, boolean and other types support has been introduced in parameters like puppet parameters.
```

Comment:
I would change `like puppet parameters` to `, it now supports the same types as smart variables and smart class parameters`.